index.html

```diff
@@ -0,0 +1,253 @@
+                                been
+                                the industry's standard</p>
+                            <div class="luang">
+                                <ul>
+                                    <li>html</li>
+                                    <li>bootstrap</li>
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <button class="btn btn-style" type="button">See Project</button>
+                </div>
+                <div class="container-poppup">
+                    <div class="pupopp">
+                        <div class="section-art">
+                            <h2>Profesional Art Printing Data</h2>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                                been
+                                the industry's standard</p>
+                            <div class="luang">
+                                <ul>
+                                    <li>html</li>
+                                    <li>bootstrap</li>
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <button class="btn btn-style" type="button">See Project</button>
+                </div>
+                <div class="container-poppup">
+                    <div class="pupopp">
+                        <div class="section-art">
+                            <h2>Profesional Art Printing Data</h2>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                                been
+                                the industry's standard</p>
+                            <div class="luang">
+                                <ul>
+                                    <li>html</li>
+                                    <li>bootstrap</li>
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <button class="btn btn-style" type="button">See Project</button>
+                </div>
+                <div class="container-poppup">
+                    <div class="pupopp">
+                        <div class="section-art">
+                            <h2>Profesional Art Printing Data</h2>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                                been
+                                the industry's standard</p>
+                            <div class="luang">
```

Comment:
> Hi @Regiss05,
> 
> **Good job so far!** There are some issues that you still need to work on to go to the next project but you are almost there! ![clapping](https://user-images.githubusercontent.com/38294716/153888587-3e47b8be-12ea-47db-a3c2-33122e091533.gif)
> 
> ### **To highlight 💯**
> * Good commit message ✔️
> * No Linter errors ✔️
> * Follow Github flow ✔️
> 
> ### **Required Changes ♻️**
> _Check the comments under the review._
> 
> Optional suggestions Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me on @Regiss05 in your question so I can receive the notification.
> 
> _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

Beautiful summary 💪🏿

Reply:
- [ ] Kindly add a  reason why  you suggest this change also use "kindly" or "please"  so your comment sounds polite :hugs

Reply:
Thank you @prolajumokeoni  let me change

Reply:
Please can you put a meaningful class to not disturb you when you will make changes 🤗